tests/ci/slow_tests_diffusers.sh

```diff
@@ -2,6 +2,7 @@
 
 python -m pip install --upgrade pip
 export RUN_SLOW=true
+huggingface-cli login --token $1
```

Comment:
There are other places that call this script (e.g. Makefile).  We should not expect parameters to be passed to this script.  Instead, we should just assume the HF_TOKEN has been configured at a higher level (i.e. by the user/caller/ci).

Reply:
oops, sorry.  I got it backwards.  The script calls the Makefile.